src/test/java/org/alfresco/api/YamlToJson.java

```diff
@@ -97,6 +103,10 @@ public static void main(String[] args) {
             swagger = parseSwaggerDef(new File(rootPath + MODEL_DEFINITION), MODEL_DEFINITION_TITLE);
             Json.mapper().writeValue(new File(rootPath + MODEL_JSON_DEFINITION), swagger);
 
+            //Event Gateway
+            swagger = parseSwaggerDef(new File(rootPath + EVENT_GATEWAY_JSON_DEFINITION), EVENT_GATEWAY_DEFINITION_TITLE);
```

Comment:
Shouldn't it be referencing to the yaml? I mean:
```suggestion
            swagger = parseSwaggerDef(new File(rootPath + EVENT_GATEWAY_DEFINITION), EVENT_GATEWAY_DEFINITION_TITLE);
```

Reply:
Yeap, good catch